erpnext/payroll/doctype/income_tax_slab/income_tax_slab.json

```diff
@@ -90,11 +91,20 @@
    "fieldtype": "Table",
    "label": "Other Taxes and Charges",
    "options": "Income Tax Slab Other Charges"
+  },
+  {
+   "default": "Company:company:default_currency",
+   "fieldname": "currency",
```

Comment:
not required

Reply:
Tax should be in the employee's currency. That's what was discussed.